packages/web-components/fast-components/src/slider-label/slider-label.ts

```diff
@@ -54,13 +64,17 @@ export class SliderLabel extends FASTElement {
         if (!this.isSliderConfig(this.parentNode)) {
             this.config = defaultConfig;
         } else {
-            const { min, max, direction, orientation } = this
+            const { min, max, direction, orientation, disabled } = this
```

Comment:
@marjonlynch We still plan to follow up PR with some observation of the parent so that labels will update as orientation, direction, disabled, etc. change right?

Reply:
@EisenbergEffect yes, I needed to get on radio component before finishing this one up. Radio is almost ready for PR and then I will get back on slider unless priorities dictate otherwise.